node-config-docsgen/action.yml

```diff
@@ -0,0 +1,159 @@
+name: "Generate Configuration Documentation"
+description: "Generate and upload Stacks Core configuration documentation"
+branding:
+  icon: "file-text"
+  color: "blue"
+
+inputs:
+  output:
+    description: "Output file for generated documentation"
+    required: false
+    default: "./node-parameters.md"
+  min-doc-size:
+    description: "Minimum documentation size in bytes"
+    required: false
+    default: "50000"
```

Comment:
is this something statically known? i see how it's used later and that makes sense - but i'm curious where the value is coming from